esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/compare/bin/compare_task_runner.sh

```diff
@@ -10,4 +10,4 @@ KGO="${KGO_ROOT_PATH}/recipe_${KGO_METRIC}*"
 COMPARE_SCRIPT="${ESMVALTOOL_DIR}/esmvaltool/utils/testing/regression/compare.py"
 
 # Run the compare script.
-python "${COMPARE_SCRIPT}" ${FOLDER_NAME} ${KGO}
+python "${COMPARE_SCRIPT}" ${KGO} ${FOLDER_NAME}
```

Comment:
What was the purpose of this change?

Reply:
To conform to the compare script:

https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/utils/testing/regression/compare.py
line 296:
def compare(reference_dir: Optional[Path], current_dir: Path,
            verbose: bool) -> bool:
    """Compare a recipe run to a reference run.

Reply:
And this also aligns with the documentation: 
https://docs.esmvaltool.org/en/latest/utils.html#comparing-recipe-runs

"where the first argument is a reference run or a directory containing such runs and the second and following arguments are directories with runs to compare to the reference run(s)."

Reply:
Many thanks for checking this so thoroughly, @KatherineTomkins :)